src/frontend/src/components/SEO.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import useSiteMetadata from '../hooks/use-site-metadata';
+
+function SEO({ description_, lang, meta, title_ }) {
+  const { title, description } = useSiteMetadata();
```

Comment:
If you want to avoid this `title` vs. `title_` stuff, you can do:

```js
const siteMetadata = useSiteMetadata();
...
<Helmet
  title={title || siteMetadata.title}
  ...
```

Same for description.  I think that's cleaner, since you can't reuse those names with eslint.

Reply:
Changes made